app/templates/components/to-review-project-card.hbs

```diff
@@ -20,16 +20,20 @@
       </div>
     </div>
     <div class="cell medium-8 large-auto">
-      {{!-- TODO: make conditional -- if hearingsSubmittedForProject OR opted out of hearings --}}
+      {{#if (or hearingsSubmitted hearingsWaived)}}
```

Comment:
This is the sorta thing I'd also just go ahead and make computed on the controller, with a human-readable name